### PR TITLE
December Cloud Release Notes

### PR DESCRIPTION
**Pull request by gemma-shay**

### CODE REVIEW

File: releases/cloud.md [hunk 1]
```diff
@@ -15,9 +15,25 @@ Get future release notes emailed to you:
...
 {{site.data.alerts.callout_version}}
-As of November 7, 2022, {{ site.data.products.serverless }} clusters are running CockroachDB [v22.1.10](v22.1.html#v22-1-10) and new {{ site.data.products.dedicated }} clusters are running CockroachDB [v22.1.10](v22.1.html#v22-1-10).
+As of December 5, 2022, {{ site.data.products.serverless }} clusters are running CockroachDB [v22.1.10](v22.1.html#v22-1-10) and new {{ site.data.products.dedicated }} clusters are running CockroachDB [v22.1.10](v22.1.html#v22-1-10).
 {{site.data.alerts.end}}
 
+## December 5, 2022
+
+<h3> Console changes </h3>
+
+- {{ site.data.products.serverless }} clusters now have a [**Metrics** page](../{{site.versions["stable"]}}/ui-custom-chart-debug-page.html) in the Console with charts to **Monitor SQL Activity** and **Identify SQL Problems**.
```
Comment (kernfeld-cockroach) [inline releases/cloud.md:25]: I thought this already existed.

File: releases/cloud.md [hunk 2]
```diff
@@ -15,9 +15,25 @@ Get future release notes emailed to you:
...
+- The **Last used** column on the [**Table Details** page](../cockroachcloud/databases-page.html) now uses the UTC timezone.
+
+<h3> Bug fixes </h3>
+
+- Trial coupon limits for {{ site.data.products.dedicated }} clusters' storage and compute are now enforced in the [**Edit cluster**](../cockroachcloud/cluster-management.html) dialog.
+- Fixed a bug where [backups](../cockroachcloud/backups-page.html) shown for a particular day included backups for midnight on the following day.
+- Fixed a bug  on the [**Databases page**](../cockroachcloud/databases-page.html) where the number of index recommendations displayed for a database was inconsistent with the actual number of index recommendations for the database.
+- Fixed a bug that could break the [**Databases page**](../cockroachcloud/databases-page.html) when fetching index usage statistics for databases.
+- Fixed a bug that caused the {{ site.data.products.serverless }} [**Cost estimator**](../cockroachcloud/serverless-cluster-management.html#estimate-usage-cost) to display incorrect estimates.
```
Comment (kernfeld-cockroach) [inline releases/cloud.md:35]: To confirm, did we fix this bug or did we disable the cost estimator?

**Review by jramos-crl — APPROVED** [review]
LGTM!

**Review by thtruo — COMMENTED** [review]
@gemma-shay could you remind me again if we are open to calling out "Preview" features? I realize that we haven't made a mention of the new Log Export functionality that's available for select Dedicated customers (they'll need to contact their Cockroach Labs account team to get access to it, it's behind a feature flag). If we're not opposed to it, I'd like to see a mention of Log Export https://www.cockroachlabs.com/docs/cockroachcloud/export-logs.html

Thanks!

**Review by emily-horing — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15716)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @gemma-shay, @kernfeld-cockroach, and @kevin-v-ngo)

___
*[`releases/cloud.md` line 35 at r2](https://reviewable.io/reviews/cockroachdb/docs/15716#-NI9Y7H7WfDv50TfQtrD-r2-35:-NIE1xQx7NAmLrWvtvuB:bvax5dw) ([raw file](https://github.com/cockroachdb/docs/blob/b5d7f71907dcbf3588af583eda27550cc91a4661/releases/cloud.md#L35)):*
<details><summary><i>Previously, kernfeld-cockroach (Paul Kernfeld) wrote…</i></summary><blockquote>

To confirm, did we fix this bug or did we disable the cost estimator?
</blockquote></details>

This still hasn't been turned back on though the fix is in progress. Perhaps by the 5th it will be back on?

**Review by emily-horing — COMMENTED** [review]
Open q about cost estimator but otherwise LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15716#-:-NIE25us26fuIzdsXmU7:b5pv9z3)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @gemma-shay, @kernfeld-cockroach, and @kevin-v-ngo)

**Review by emily-horing — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15716)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @gemma-shay, @kernfeld-cockroach, and @kevin-v-ngo)

___
*[`releases/cloud.md` line 35 at r2](https://reviewable.io/reviews/cockroachdb/docs/15716#-NI9Y7H7WfDv50TfQtrD-r2-35:-NIE2jjz9wAxVatANpQ6:b21o1kn) ([raw file](https://github.com/cockroachdb/docs/blob/b5d7f71907dcbf3588af583eda27550cc91a4661/releases/cloud.md#L35)):*
<details><summary><i>Previously, emily-horing wrote…</i></summary><blockquote>

This still hasn't been turned back on though the fix is in progress. Perhaps by the 5th it will be back on?
</blockquote></details>

Talked to Nathan. It will be turned on next week but depending on when the next release is may not be on on the 5th. I think it's ok to keep in though.

File: releases/cloud.md (hunk 1 above)
Comment (gemma-shay) [reply releases/cloud.md:25]: It never had a release note, so just adding one now. I think it hasn't existed for long.

File: releases/cloud.md (hunk 2 above)
Comment (gemma-shay) [reply releases/cloud.md:35]: Hm, looks like it was actually disabled. Good catch, I'll change the note.

File: releases/cloud.md (hunk 1 above)
Comment (kernfeld-cockroach) [reply releases/cloud.md:25]: I remember that I including these pages in a [document](https://docs.google.com/document/d/1TK5JZaS8kNTqq_wnQWbzT9MxXaDthV5qboFlCx6CuVU/edit#) that I worked on around September/October, so I don't think they're that new. Would you mind confirming the age of these pages?

**Comment by gemma-shay** [comment]
@thtruo we don't include preview features automatically but happy to add by request! hopefully process will be changing soon

File: releases/cloud.md (hunk 1 above)
Comment (gemma-shay) [reply releases/cloud.md:25]: @thtruo @andf-crl do you know when the metrics page was released for serverless?

Comment (thtruo) [reply releases/cloud.md:25]: @gemma-shay Custom metrics were introduced in September for Serverless

Comment (andf-crl) [reply releases/cloud.md:25]: #[14998](https://github.com/cockroachdb/docs/pull/14998/files) saw the [serverless metrics list](https://www.cockroachlabs.com/docs/stable/ui-custom-chart-debug-page.html?filters=metric-names-serverless); merged from [this PR](https://github.com/cockroachdb/docs/pull/14998/files) published.

This is somewhat similar to the note for log export, which was also introduced to CC previous to the Dec 5 CC release (Aug / Sep '22). I'm not sure how CC does it: but for CRDB I'd only include release notes for changes specific to the version the release applies to, even if previous RNs failed to capture a CC change.

Comment (gemma-shay) [reply releases/cloud.md:25]: We could move it to the September notes, leave it here, or remove the note. I would prefer to leave it in December so users don't miss it, but open to thoughts.

Comment (gemma-shay) [reply releases/cloud.md:25]: @andf-crl whoops I commented before I saw your reply. We don't really attach notes to versions in CC; it's just chronological.

**Review by rmloveland — APPROVED** [review]
LGTM!

**Pull request merged**